scripts/flux.js

```diff
@@ -0,0 +1,50 @@
+export default class Flux {
+
+  constructor(components, events) {
+    this.store = components.store;
+    this.auth = components.auth;
+    this.events = events;
+
+    this.events.on('auth:login', this.onLogin.bind(this));
+    this.events.on('action:configure', this.onConfigure.bind(this));
+    this.events.on('action:create', this.onCreate.bind(this));
+    this.events.on('action:update', this.onUpdate.bind(this));
+    this.events.on('action:delete', this.onDelete.bind(this));
+    this.events.on('action:sync', this.onSync.bind(this));
+  }
+
+  on(events, callback) {
+    const names = events.split(/\s*,\s*/);
+    names.forEach(event => {this.events.on(event, callback)});
+  }
+
+  dispatch(name, data) {
+    this.events.emit(name, data);
+  }
+
+  onConfigure(config) {
+    this.auth.configure(config);
+    this.auth.authenticate();
```

Comment:
Subscribing to an event seems more consistent with the rest of the code, I'd vote for that.